### PR TITLE
[TestSuite] Add test suite for cephadm regression tests on upstream builds

### PR DESCRIPTION
**Pull request by vamahaja**

### CODE REVIEW

File: suites/quincy/upstream/tier-1-cephadm-basic-regression.yaml
Comment (sunilkumarn417) [inline suites/quincy/upstream/tier-1-cephadm-basic-regression.yaml:270]: I  think, we dont have cephadm-ansible in upstream repo rpms. 
https://1.chacra.ceph.com/r/ceph/main/8e459c55f4d7afef71d5cdfcba01b9583fb3ff3e/centos/9/flavors/default/

Comment (vamahaja) [reply suites/quincy/upstream/tier-1-cephadm-basic-regression.yaml:270]: thanks @sunilkumarn417 for pointing this out

**Review by pranavprakash20 — APPROVED** [review]
LGTM

**Review by MohitBis — DISMISSED** [review]
LGTM

**Comment by vamahaja** [comment]
Test Run - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-f0o2v/

One failed TC - `Executes RGW, RBD and FS operations` with error 
```
2023-01-13 14:55:31,023 - ERROR - cephci.ceph.ceph.py:1576 - Error 1 during cmd, timeout 600
2023-01-13 14:55:31,025 - ERROR - cephci.ceph.ceph.py:1577 - Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)')

2023-01-13 14:55:31,025 - INFO - cephci.tests.cephfs.cephfs_basic_tests.py:64 - ceph auth get-or-create client.ceph-ci-vamahaja-wc7s6-u2ubr8-node5 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-ci-vamahaja-wc7s6-u2ubr8-node5.keyring Error:  Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)')
 10.0.210.160
2023-01-13 14:55:31,026 - INFO - cephci.tests.cephfs.cephfs_basic_tests.py:65 - Traceback (most recent call last):
  File "/home/jenkins-build/workspace/rhceph-test-executor/tests/cephfs/cephfs_basic_tests.py", line 21, in run
    fs_util.auth_list(clients)
  File "/home/jenkins-build/workspace/rhceph-test-executor/tests/cephfs/cephfs_utilsV1.py", line 172, in auth_list
    cmd=f"ceph auth get-or-create client.{client.node.hostname}"
  File "/home/jenkins-build/workspace/rhceph-test-executor/ceph/ceph.py", line 2109, in exec_command
    return self.node.exec_command(cmd=cmd, **kw)
  File "/home/jenkins-build/workspace/rhceph-test-executor/ceph/ceph.py", line 1579, in exec_command
    f"{kw['cmd']} Error:  {str(stderr)} {str(self.ip_address)}"
ceph.ceph.CommandFailed: ceph auth get-or-create client.ceph-ci-vamahaja-wc7s6-u2ubr8-node5 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-ci-vamahaja-wc7s6-u2ubr8-node5.keyring Error:  Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)')
 10.0.210.160
```

**Comment by sunilkumarn417** [comment]
> Test Run - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-f0o2v/
> 
> One failed TC - `Executes RGW, RBD and FS operations` with error
> 
> ```
> 2023-01-13 14:55:31,023 - ERROR - cephci.ceph.ceph.py:1576 - Error 1 during cmd, timeout 600
> 2023-01-13 14:55:31,025 - ERROR - cephci.ceph.ceph.py:1577 - Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)')
> 
> 2023-01-13 14:55:31,025 - INFO - cephci.tests.cephfs.cephfs_basic_tests.py:64 - ceph auth get-or-create client.ceph-ci-vamahaja-wc7s6-u2ubr8-node5 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-ci-vamahaja-wc7s6-u2ubr8-node5.keyring Error:  Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)')
>  10.0.210.160
> 2023-01-13 14:55:31,026 - INFO - cephci.tests.cephfs.cephfs_basic_tests.py:65 - Traceback (most recent call last):
>   File "/home/jenkins-build/workspace/rhceph-test-executor/tests/cephfs/cephfs_basic_tests.py", line 21, in run
>     fs_util.auth_list(clients)
>   File "/home/jenkins-build/workspace/rhceph-test-executor/tests/cephfs/cephfs_utilsV1.py", line 172, in auth_list
>     cmd=f"ceph auth get-or-create client.{client.node.hostname}"
>   File "/home/jenkins-build/workspace/rhceph-test-executor/ceph/ceph.py", line 2109, in exec_command
>     return self.node.exec_command(cmd=cmd, **kw)
>   File "/home/jenkins-build/workspace/rhceph-test-executor/ceph/ceph.py", line 1579, in exec_command
>     f"{kw['cmd']} Error:  {str(stderr)} {str(self.ip_address)}"
> ceph.ceph.CommandFailed: ceph auth get-or-create client.ceph-ci-vamahaja-wc7s6-u2ubr8-node5 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-ci-vamahaja-wc7s6-u2ubr8-node5.keyring Error:  Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)')
>  10.0.210.160
> ```

Hey @vamahaja ,

Let's reach out to respective people and try to fix this issues before we merge in.

**Comment by vamahaja** [comment]
@sunilkumarn417 there will be separate patch to fix failure for TC. Better not hold this PR as won't affect pipeline.

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by pranavprakash20 — APPROVED** [review]
Lgtm

**Review by sunilkumarn417 — APPROVED** [review]

**Pull request merged**
